### PR TITLE
Create Finnish translation

### PR DESCRIPTION
**Pull request by direc85**
- Create Finnish translation
- Add .gitignore (for those not doing shadow builds)

### CODE REVIEW

**Comment by direc85** [comment]
It looks like I missed something, because the app isn't translated when I install and launch my build...

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cmake_install.cmake
+CMakeCache.txt
+CMakeFiles/
+documentation.list
+icons/
```
Comment (mentaljam) [inline .gitignore:5]: This directory contains sources too

**Review by mentaljam — COMMENTED** [review]
Not listed entries:

- Sailfish build system produces some hidden directories in the build root
- When packaging produces some `debug*.list` files

In general CMake doesn't support not-shadow builds good. Also I'm curious why do you prefer build projects in place?

File: .gitignore (hunk 1 above)
Comment (direc85) [reply .gitignore:5]: Thanks for pointing these out.

I'm not that familiar with `sfdk` yet (stubborn Qt Creator click-to-run guy) so I guess I have messed something up. I'll check the ignored files again with a clean pull.

**Comment by rinigus** [comment]
Sorry for delay. I would suggest to close this PR and make a new one limited to addition of translations. Changes in CMakefile can be added, but .gitignore changes should be in a separate PR.

As it is getting into conflicts on .gitignore, I suggest to close it here and resubmit with the translation. Note that we may change few things still and it maybe better to look into translations few days later.

**Comment by direc85** [comment]
Whoops, totally forgot about this one.

Sure thing, I'll close this one and try again later (with less haste, I might add) :slightly_smiling_face:

**Pull request closed without merging**